app/controllers/airlines_controller.rb

```diff
@@ -3,7 +3,8 @@ class AirlinesController < ApplicationController
 
   def index
     sorted_airlines = Airline.sort_by_rating
-    @airlines = Kaminari.paginate_array(sorted_airlines).page(params[:page]).per(24)
+    @airlines = Kaminari.paginate_array(sorted_airlines)
+                        .page(params[:page]).per(24)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 20) spaces for indenting an expression in an assignment spanning multiple lines.